recipes/opencv/4.x/conanfile.py

```diff
@@ -342,7 +402,16 @@ def gtk():
             return ["gtk::gtk"] if self.options.get_safe("with_gtk") else []
 
         def freetype():
-            return ["freetype::freetype"] if self.options.contrib else []
+            return ["freetype::freetype"] if self.options.with_freetype else []
+
+        def harfbuzz():
+            return ["harfbuzz::harfbuzz"] if self.options.with_harfbuzz else []
+
+        def glog():
+            return ["glog::glog"] if self.options.with_glog else []
+
+        def gflags():
+            return ["gflags::gflags"] if self.options.with_gflags else []
```

Comment:
Should these have a get_safe? https://github.com/conan-io/conan-center-index/pull/4797/files#diff-8938123ce86db129790cfb63fac38df04c5cefb2bbfda9cf37a9b8925061a9cfR93-R97

Reply:
With the added `del`, you're right. I'll add them.